src/components/UserFriends/UserFriends.js

```diff
@@ -1,18 +1,46 @@
-import React, { useState } from 'react';
-import { connect } from 'react-redux';
-import mapStoreToProps from '../../redux/mapStoreToProps';
+import React, { useState } from "react";
+import { connect } from "react-redux";
+import mapStoreToProps from "../../redux/mapStoreToProps";
 
 // Basic functional component structure for React with default state
 // value setup. When making a new component be sure to replace the
 // component name TemplateFunction with the name for the new component.
 function TemplateFunction(props) {
```

Comment:
naming.
I would also suggest getting away from copy and pasting the template.
You can have it open next to your new file for reference, but practice writing from scratch.